src/lib/ngsi10/QueryContextRequestVector.cpp

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Yeah, I was thinking about this also.
I honestly don't know in which directory these four new files belong ...
orionTypes perhaps ... ?